test/everypolitician/popolo/organization_test.rb

```diff
@@ -1,10 +1,20 @@
 require 'test_helper'
 
 class OrganizationTest < Minitest::Test
-  def test_reading_popolo_organizations
-    popolo = Everypolitician::Popolo::JSON.new(organizations: [{ id: '123', name: 'ACME' }])
-    assert_instance_of Everypolitician::Popolo::Organizations, popolo.organizations
-    organization = popolo.organizations.first
+  def organizations
+    @organizations ||= Everypolitician::Popolo.read('test/fixtures/turkey-ep-popolo-v1.0.json').organizations
+  end
+
+  def organization
+    @organization ||= organizations.select { |o| o.id == 'MHP' }.first
```

Comment:
I think you'd be better always going through the published interface for `find_by` or `where` instead of rolling your own `select`s